### PR TITLE
Priority fix

### LINKED ISSUE #427: Priority for matrix operators is broken
**The version I use**: AngouriMath.Interactive 0.0.0-master-1621280613-564474d

**Unexpected behaviour or bug**: `|*` seems to have lower priority than `|+`

UPD: if you want to close this issue, make sure to add tests which examine all cases where the priority of operators might work incorrectly.

### PR DESCRIPTION
**Pull request by WhiteBlackGoose**
Resolves #427

### CODE REVIEW

**Comment by WhiteBlackGoose** [comment]
@Happypig375 idk if you foresaw so or not, but your form of operator for matrices (like `+.`) was better than mine (`|+`) in a sense, that they're ordered in a correct priority.

**Comment by Happypig375** [comment]
🦆

File: Sources/Tests/FSharpWrapperUnitTests/MatricesTest.fs [hunk 1]
```diff
@@ -34,32 +34,156 @@ let ``M*M`` () =
...
+    Assert.Equal<Entity.Matrix>(a **. b -. c, (a **. b) -. c) 
+
+[<Fact>]
+let ``Priority -.**.`` () =
+    Assert.Equal<Entity.Matrix>(a -. b **. c, a -. (b **. c))
+    
+    
+[<Fact>]
+let ``Priority **.*.`` () =
+    Assert.Equal<Entity.Matrix>(a **. b +. c, (a **. b) +. c)
```
Comment (Happypig375) [inline Sources/Tests/FSharpWrapperUnitTests/MatricesTest.fs:119]: :duck:

File: Sources/Tests/FSharpWrapperUnitTests/Functions.Matrices.fs [hunk 2]
```diff
@@ -16,13 +16,13 @@ let ``Matrix test 1`` () = Assert.Equal(parsed "[[1, 2], [2, 3], [3, 4]]", matri
 [<Fact>]
 let ``Matrix test 2 transposed`` () = Assert.Equal(parsed "[[1, 2], [2, 3], [3, 4]]T", transposed (matrix [[1; 2]; [2; 3]; [3; 4]]))
 [<Fact>]
-let ``Tensor product 1`` () = Assert.Equal(parsed "[a c, a d, b c, b d]", (asMatrix "[a, b]") *** (asMatrix "[c, d]"))
+let ``Tensor product 1`` () = Assert.Equal(parsed "[a c, a d, b c, b d]", (asMatrix "[a, b]") ***. (asMatrix "[c, d]"))
```
Comment (Happypig375) [inline Sources/Tests/FSharpWrapperUnitTests/Functions.Matrices.fs:19]: Parentheses not needed

File: Sources/Tests/FSharpWrapperUnitTests/MatricesTest.fs (hunk 1 above)
Comment (WhiteBlackGoose) [reply Sources/Tests/FSharpWrapperUnitTests/MatricesTest.fs:119]: 🦆

**Pull request merged**